src/plugin/events.c

```diff
@@ -138,6 +139,8 @@ _purple_events_init(PurplePlugin *plugin)
                            purple_value_new(PURPLE_TYPE_STRING), purple_value_new(PURPLE_TYPE_STRING), purple_value_new(PURPLE_TYPE_STRING), purple_value_new(PURPLE_TYPE_STRING));
     purple_signal_register(plugin, "conversation-got-focus", purple_marshal_VOID__POINTER_POINTER_POINTER_POINTER, NULL, 1,
                            purple_value_new(PURPLE_TYPE_SUBTYPE, PURPLE_SUBTYPE_CONVERSATION));
+    purple_signal_register(plugin, "authorization-requested", purple_marshal_VOID__POINTER_POINTER_POINTER, NULL, 3,
```

Comment:
The event is specific to a buddy, so it must be prefixed with `user_` as the others, and sorted before the `conversation-got-focus` one, which is an internal ones to dismiss notifications.